src/includes/classes/Plugin.php

```diff
@@ -490,6 +490,12 @@ public function setup()
         /*[/pro]*/
 
         /*[pro strip-from="lite"]*/
+        if ($this->options['when_logged_in'] === '1' && $this->applyWpFilters(GLOBAL_NS.'_when_logged_in_no_admin_bar', true)) {
+          show_admin_bar(FALSE); // Prevent admin bar from being cached.
```

Comment:
@jaswsinc It might be better to use `add_filter('show_admin_bar', '__return_false');` here instead of `show_admin_bar(FALSE)`, as the latter hides the Admin Bar from the front-end _and_ the backend (Dashboard). Since ZenCache is only affecting the front-end of the site, there's no need to also get rid of it on the Dashboard.